students/BrandonHenson/Lesson3/mailroom.py

```diff
@@ -0,0 +1,82 @@
+# Brandon Henson
+# 4/8/18
+# Lesson 03 mailroom.py
+
+donor_history = [
+            ['Brandon Henson', 1005.49, 3116.72, 5200],
+            ['Alicia Henson', 21.47, 1500],
+            ['Michael Green', 2400.54],
+            ['Brandon Henson Jr', 355.42, 579.31],
+            ['Kaiya Henson', 636.9, 850.13, 125.23]
+    ]
+name_list = []
```

Comment:
Instead of building a second list here for donor lookups you might want to define a function that takes a name and returns the donor/donation list for that donor if it exists.